lib/statsd_middleware.rb

```diff
@@ -3,6 +3,52 @@
 class StatsdMiddleware
   STATUS_KEY   = 'api.rack.request'
   DURATION_KEY = 'api.rack.request.duration'
+  SOURCE_APP_NAMES = %w[
```

Comment:
how do we intend for FE engineers to know to keep this list up to date?  Can you add a comment, so people in the future know that this should be kept up to date manually and FE engineers

Reply:
Good question. Some thoughts and stuff:

- added logic to warn to sentry when an unrecognized `Source-App-Name` header comes in
- add instructional comment on how to regenerate 
- if we're out of date on the whitelist it isnt _that big of a deal_, it will result in some small holes in our metrics